pkg/envoy/cell.go

```diff
@@ -214,9 +216,10 @@ type versionCheckParams struct {
 	cell.In
 
 	Lifecycle        cell.Lifecycle
+	Slog             *slog.Logger
```

Comment:
not a fan of having two loggers injected here and in other places (slog & logrus). Is it up to the owners to clean this up afterwards?

Reply:
Thanks for the explanation David!